packages/driver/src/cypress/runner.ts

```diff
@@ -736,7 +724,7 @@ const normalize = (runnable, tests, initialTests, onLogsById, getRunnableId, get
       suite.suites = []
 
       normalizedSuite.suites = _.compact(_.map(suiteSuites, (childSuite) => {
-        const normalizedChildSuite = normalize(childSuite, tests, initialTests, onLogsById, getRunnableId, getHookId, getOnlyTestId, getOnlySuiteId, createEmptyOnlyTest)
+        const normalizedChildSuite = normalize(childSuite, tests, previouslyRanTests, onLogsById, getRunnableId, getHookId, getOnlyTestId, getOnlySuiteId, createEmptyOnlyTest)
```

Comment:
Thanks for renaming this - previouslyRanTests is much clearer about what's going on and why tests would be in there.